app/models/shipit/repository.rb

```diff
@@ -66,16 +66,24 @@ def github_repo_name
       [owner, name].join('/')
     end
 
+    def github_api
+      github_app_instance.api
+    end
+
+    def github_app_instance
```

Comment:
The `_instance` suffix is ugly. Why is it needed?

Reply:
It's not :) No issues in removing it.